packages/react-sdk/src/components/ImportWhiteboardDialog/pdfImportUtils.ts

```diff
@@ -22,7 +22,13 @@ import {
 } from 'pdfjs-dist';
 import * as pdfJSWorker from 'pdfjs-dist/build/pdf.worker.mjs?url';
 import { RenderParameters } from 'pdfjs-dist/types/src/display/api';
-import { whiteboardHeight, whiteboardWidth } from '../Whiteboard/constants';
+import {
+  frameHeight,
+  frameWidth,
+  infiniteCanvasMode,
+  whiteboardHeight,
+  whiteboardWidth,
+} from '../Whiteboard/constants';
```

Comment:
Suggest to put `frameHeight`, `frameWidth` to `Whiteboard` `index.ts` file and import them using `from '../Whiteboard'`. Other constants with exception of `infiniteCanvasMode` are there, maybe apply the same change for `infiniteCanvasMode` const as well.

Reply:
Its a reexport in there actually but I add it :)